### PR TITLE
Minor fix and re-run the generation tool

### PR DESCRIPTION
**Pull request by michaelawyu**
fix: update the generation tool to use the new schema format.

Re-run the generation to include the latest event types.

### CODE REVIEW

File: src/https:/googleapis/github/io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData/json/RemoteConfigEventData.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+from typing import Optional, Union
```
Comment (grant) [inline src/https:/googleapis/github/io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData/json/RemoteConfigEventData.py:1]: Do we know what happened to the header here?

**Review by grant — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -5,8 +5,22 @@ At this moment the following types are available:
...
+| google.events.cloud.scheduler.v1 | SchedulerJobData | Scheduler job data. |
+| google.events.cloud.pubsub.v1 | MessagePublishedData | The data received in an event when a message is published to a topic. |
+| google.events.firebase.database.v1 | ReferenceEventData | The data within all Firebase Real Time Database reference events. |
+| google.events.firebase.remoteconfig.v1 | RemoteConfigEventData | The data within all Firebase Remote Config events. |
+| google.events.firebase.auth.v1 | AuthEventData | The data within all Firebase Auth events |
+| google.events.cloud.storage.v1 | StorageObjectData | An object within Google Cloud Storage. |
+| google.events.cloud.firestore.v1 | DocumentEventData | The data within all Firestore document events. |
+| google.events.firebase.analytics.v1 | AnalyticsLogData | The data within Firebase Analytics log events. |
+| google.events.cloud.audit.v1 | LogEntryData | Generic log entry, used as a wrapper for Cloud Audit Logs in events.
+ This is copied from
```
Comment (grant) [inline README.md:17]: I don't think this table has the right contents here.

File: src/https:/googleapis/github/io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData/json/RemoteConfigEventData.py (hunk 1 above)
Comment (michaelawyu) [reply src/https:/googleapis/github/io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData/json/RemoteConfigEventData.py:1]: Sorry I am not sure I understand the question here. By header you mean the copyright disclaimer?

File: README.md (hunk 2 above)
Comment (michaelawyu) [reply README.md:17]: Do you mean the line breaks? I was not aware that we have line breaks in the description field now; will update the PR.

Comment (michaelawyu) [reply README.md]: The description is now formatted correctly.

File: src/https:/googleapis/github/io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData/json/RemoteConfigEventData.py (hunk 1 above)
Comment (grant) [reply src/https:/googleapis/github/io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData/json/RemoteConfigEventData.py:1]: Yes, like the sister `__init__.py` file. The `Copyright 2020 Google LLC...` header. Would be nice to have.

**Review by grant — COMMENTED** [review]
(Replied to comment, PR is still approved)

File: src/https:/googleapis/github/io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData/json/RemoteConfigEventData.py (hunk 1 above)
Comment (michaelawyu) [reply src/https:/googleapis/github/io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData/json/RemoteConfigEventData.py]: Done.

**Pull request merged**